### ISSUE TITLE
Lamdatest giving more detailed audits on contrast as compared to lighthouse

### ISSUE THREAD

**Issue by Shivam93937** [GoogleChrome/lighthouse#16517]
### FAQ

- [x] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [x] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).
- [x] Yes, my issue is not answered by [other FAQs](https://github.com/GoogleChrome/lighthouse#faq).

### URL

play.geforcenow.com

### What happened?

Lamdatest extension is giving more detailed audits as compared to Lighthouse audits on accessibility issues.

Lamdatest : 
![Image](https://github.com/user-attachments/assets/3baa0085-5ece-40d9-9a1f-bc9310331d2e)

Lighthouse:
![Image](https://github.com/user-attachments/assets/63172e05-4334-4c90-a242-aefb511ab5e6)

Mainly, the contrast audits are passing in lighthouse but failing on lambdatest. 
Both of them seem to use the same guidelines, WCAG.

**I filed this issue with axe-devtools, they asked me to report it on lighthouse instead.
https://github.com/dequelabs/axe-core/issues/4778#issuecomment-2919001360**

### What did you expect?

I wanted to see the full detailed list of accessibility audits that were missing on lighthouse.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

12.5.1

### Chrome Version

136.0.7103.114

### Relevant log output

```sh

```

**Comment by Shivam93937** [comment]
Hi, any follow ups on this?
